### ISSUE TITLE
Error: EEXIST: file already exists node_modules/@size-limit/.cache/size-limit

### ISSUE THREAD

**Issue by pustovalov** [ai/size-limit#139]
Sometimes such an error occurs on the CI:

```
yarn run v1.17.3
$ yarn run size-limit
$ /home/circleci/main/node_modules/.bin/size-limit
(node:542) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 exit listeners added to [process]. Use emitter.setMaxListeners() to increase limit
(node:542) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 SIGINT listeners added to [process]. Use emitter.setMaxListeners() to increase limit
(node:542) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 SIGTERM listeners added to [process]. Use emitter.setMaxListeners() to increase limit
(node:542) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 SIGHUP listeners added to [process]. Use emitter.setMaxListeners() to increase limit
 ERROR  Error: EEXIST: file already exists, mkdir '/home/circleci/main/node_modules/@size-limit/.cache/size-limit'
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Exited with code 1
```

`.size-limit.json`:
```
[
  {
    "path": "public/packs-test/vendors-*.js",
    "limit": "680 KB"
  },
  {
    "path": "public/packs-test/lite/lite-*.js",
    "limit": "100 KB"
  },
  {
    "path": "public/packs-test/common-*.js",
    "limit": "10 KB"
  },
  {
    "path": "public/packs-test/sites/sites_spa-*.js",
    "limit": "25 KB"
  },
  {
    "path": "public/packs-test/campaign_rules-*.js",
    "limit": "45 KB"
  },
  {
    "path": "public/packs-test/font_uploader-*.js",
    "limit": "10 KB"
  },
  {
    "path": "public/packs-test/auth-*.js",
    "limit": "10 KB"
  },
  {
    "path": "public/packs-test/campaign_copy-*.js",
    "limit": "5 KB"
  },
  {
    "path": "public/packs-test/global_ajax_event_handlers-*.js",
    "limit": "5 KB"
  },
  {
    "path": "public/packs-test/info_sidebar-*.js",
    "limit": "5 KB"
  },
  {
    "path": "public/packs-test/multiple_campaign_editor-*.js",
    "limit": "15 KB"
  },
  {
    "path": "public/packs-test/accounts/accounts_edit-*.js",
    "limit": "25 KB"
  },
  {
    "path": "public/packs-test/bundle_ab_tests-*.js",
    "limit": "15 KB"
  },
  {
    "path": "public/packs-test/bundle_editor-*.js",
    "limit": "200 KB"
  },
  {
    "path": "public/packs-test/bundle_campaign_ab_tests-*.js",
    "limit": "15 KB"
  }
]
```

`Node`: 12.8.1
`@size-limit/preset-app`: 2.1.1

**Comment by ai** [comment]
How do you run Size Limit?

**Comment by ai** [comment]
Fixed https://github.com/ai/size-limit/commit/05e95b8ef84247795ebcf7950e17002938447540

**Comment by ai** [comment]
Released in 2.1.5

**Comment by pustovalov** [comment]
thx, I will try the new version
